scripts/startup/start_panoptes_in_tmux.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash -ex
 
+if [[ -z "$BASH_VERSION" ]] ; then
```

Comment:
If you have the bash shebang then why is this necessary?

Reply:
Here I probably don't, but I was struggling to diagnose the failures. If I debugged this correctly, it came down to the `tmux new-window` command using `$SHELL` and `bash --login` not setting `$SHELL`.

`start_panoptes_in_tmux.sh` is executed by bash because of the shebang, but the windows (i.e. virtual terminals) that it creates using `tmux new-window` don't have a command specified, so just run the default shell. Those windows are then sent commands that are designed to work in a bash shell with POCS, PANDIR, etc. all set, and the conda panoptes-env activated. Without the change in tmux_launch.sh, the default shell ends up being /bin/sh, which doesn't support the commands that are sent to the window.